Taskfile.yml

```
@@ -0,0 +1,36 @@
+version: "3"
+
+tasks:
+
+  docker:
+    des: Namespace for Docker-related tasks
+    aliases: [d]
+    cmds:
+      - task: docker:build
+
+  docker:build:
+    desc: Build Docker image
+    cmds:
+      - |
+        echo "local build"
+        docker buildx build \
+        --tag backend:latest \
+        .
+  
+  docker:run:
+    desc: Run Docker container. If the container already exists, it removes it before starting a new one.
+    vars:
+      CONTAINER_NAME: '13_eclaireur_public-eclaireur-public-back'
+    cmds:
+      - |
+        if [ "$(docker ps -aq -f name=^{{.CONTAINER_NAME}})$" ] ; then 
+        echo "This container already exists, removing it..."
+        docker rm -f {{.CONTAINER_NAME}} 
+        fi
+        docker run -d --name {{.CONTAINER_NAME}} \
+        --volume $(pwd)/back/data:/data \
+        backend:latest
+        echo 'Start Container {{.CONTAINER_NAME}}, to see the logs run: docker logs -f {{.CONTAINER_NAME}}'
```

nitpick: ça c'est une préférence personnelle, si je lance le docker, j'aime autant avoir les logs directements en console. Tu peux rejeter si tu préfère.

```suggestion
        docker logs -f {{.CONTAINER_NAME}}
```